infra/main.tf

```diff
@@ -0,0 +1,38 @@
+terraform {
+  required_providers {
+    prefect = {
+      source = "PrefectHQ/prefect"
+    }
+  }
+}
+
+provider "prefect" {
+  api_key = var.prefect_api_key
+  account_id = var.prefect_account_id
+}
+
+# Create staging workspace
```

Comment:
If we want to get a bit fancier we should separate these into [modules](https://developer.hashicorp.com/terraform/language/modules/develop).